### ISSUE TITLE
chair's department in abstract

### ISSUE THREAD

**Issue by amilesj** [benmarwick/huskydown#10]
Hi there,

The UW template for the abstract has the chair's department listed under their name: http://grad.uw.edu/wordpress/wp-content/uploads/sample-abstract.pdf

Is it possible to add this to huskydown?

**Comment by benmarwick** [comment]
Huskydown currently does this, here's an `index.Rmd` file I just made:

![image](https://user-images.githubusercontent.com/1262179/53443225-0e6c6680-39c0-11e9-8ab3-ac91e9a5d15f.png)

And here's the Abstract page of the PDF it produces:

![image](https://user-images.githubusercontent.com/1262179/53443254-1e844600-39c0-11e9-96a1-a7ff7eaeba08.png)

Can you show me more about what you're attempting to do so we can try to figure out why you're not seeing the chair's department listed under their name on the abstract page?

**Comment by amilesj** [comment]
Hey, thanks for the quick reply. I should have tried it out with unaltered `index.Rmd` and `template.tex` files, sorry about that. 

I've made multiple customizations along the way, so I'm sure it was a side effect of a change I made that I just noticed. I was able to get the department back in the abstract by editing the `uwthesis.cls` file directly. Again, apologies for not doing some more digging first!
